experimental/saiexperimentaldashacl.h

```diff
@@ -110,6 +110,38 @@ typedef enum _sai_dash_acl_rule_attr_t
      */
     SAI_DASH_ACL_RULE_ATTR_DASH_ACL_GROUP_ID,
 
+    /**
+     * @brief Ternary matched key dst_tag
+     *
+     * @type sai_uint32_t
```

Comment:
DASH behavioral model needs to be changed to make this attribute's type sai_object_list_t, per discussion with @prsunny

Reply:
As you know, this SAI header is automatically generated from P4. So, if we discard the bitmap design, should we also leverage a customized MACRO like `#define LIST_MATCH list`?